rerun_py/rerun_sdk/rerun/log/points.py

```diff
@@ -183,7 +186,8 @@ def log_points(
     entity_path:
         Path to the points in the space hierarchy.
     positions:
-        Nx2 or Nx3 array
+        Nx2 or Nx3 array.
+        An empty array with now dimensionality will an empty 2D points array.
```

Comment:
This makes no sense on so many level 😆

Reply:
"An empty array will be interpreted as a an empty 2D points array"?

But this seems error prone to me. Why 2D and not 3D?

Reply:
no reason really. Brought this up on Slack two weeks ago and we concluded that this is the path of least resistance for the (soon) old API

Reply:
I guess this is a damned-if-you-do, damned-if-you-don't situation, but this will be surprising for 3D point cloud users. Maybe we should warn when this happens?